### ISSUE TITLE
[BUG]Can't launch the game

### ISSUE THREAD

**Issue by arkydave** [FSGModding/FSG_Mod_Assistant#147]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Just installed Mod Manager, created  a mod folder, made it active and tried unsuccessfully to launch the game. Debug log looks like this:

2023-07-09 10:32:19 |    INFO | ModAssist Logger    : 2.8.2
2023-07-09 10:32:19 |    INFO |  - Node.js Version  : 18.15.0
2023-07-09 10:32:19 |    INFO |  - Electron Version : 25.2.0
2023-07-09 10:32:19 |    INFO |  - Chrome Version   : 114.0.5735.134
2023-07-09 10:32:19 |    INFO | translate-lib >> Starting i18n Library :: en
2023-07-09 10:32:19 |    INFO | auto-update >> Checking for update
2023-07-09 10:32:20 |    INFO | modhub-cache >> Got modHubDataCombo.json: 200
2023-07-09 10:32:21 |    INFO | auto-update >> Update for version 2.8.2 is not available (latest version: 2.8.2, downgrade is disallowed).
2023-07-09 10:32:52 |  DANGER | game-launcher >> Game launch failed Error: spawn C:\Program Files (x86)\Steam\steamapps\common\Farming Simulator 22 ENOENT!

### Expected Behavior

Expected FS22 to launch with mods list in my Addington Park mod folder.

### Environment

```markdown
- Mod Assistant Version: 2.8.2
- OS: Windows 11
- Virtualized: No
```

**Comment by jtsage** [comment]
Do you have the steam version of the game, and is the game installed in the default location?

`C:\Program Files (x86)\Steam\steamapps\common\Farming Simulator 22` is where it is looking.

**Comment by arkydave** [comment]
Yes. Here's a screenshot:



    On Sunday, July 9, 2023 at 10:58:59 AM CDT, J.T. Sage ***@***.***> wrote:  
 
 


Do you have the steam version of the game, and is the game installed in the default location?

C:\Program Files (x86)\Steam\steamapps\common\Farming Simulator 22 is where it is looking.

**Comment by jtsage** [comment]
Fwiw, the screenshot didn't come through.  You can give re-setting the game path again and see if that clears it up - I have seen this pop up a couple times before, still not totally sure what causes this.  Are you using any game arguments?

**Comment by jtsage** [comment]
So, the answer for this is to explicitly set the game path in settings - the manual setting works, the automatic guessing was leaving the game EXE name off the end.  This will be fixed for new users in 3.0.0 in a few days, but anybody who downloaded before then will have to fix it themselves.  Sorry about that.

**Comment by arkydave** [comment]
Thank you, sir. I set the game path as you suggested, and it works beautifully. Think I'm going to love having this utility. Great job.
